pokemonCards/pokemoncard_api/models.py

```diff
@@ -26,3 +26,11 @@ class PokemonCardSet(models.Model):
     def __str__(self):
         return self.name
 
+class PokemonCollection(models.Model):
+    user = models.CharField(max_length=100)
+    collectedCard = models.ForeignKey("PokemonCard", on_delete=models.CASCADE)
+    quantity = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
```

Comment:
name isn't a field on this class, you will need to pick one of the other fields for this